src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs

```diff
@@ -188,7 +188,7 @@ public static string GetPHPNamespace(this string @namespace, string prefix = "")
 
             pieces.AddRange(@namespace.Split('.').ToList());
 
-            var pascalCasePieces = pieces.Select(piece => piece.Pascalize());
+            var pascalCasePieces = pieces.Select(piece => piece.ToPascalize());
```

Comment:
old name makes more sense :)

Reply:
I agree, however old name maps directly to the inflector extension method when the "new" one maps to our own extension methods. This is the only place as far as I remember that was diverging. I think the original idea behind using our own extension method was to encapsulate inflector.

Reply:
Yeah, we should probably rename our extension method (not in this PR) to something like: `ToPascalCase()`.